pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/mergerollup/MergeRollupTaskGenerator.java

```diff
@@ -196,6 +208,7 @@ public String getTaskType() {
         // Get the bucket size and buffer
         long bucketMs =
             TimeUtils.convertPeriodToMillis(mergeConfigs.get(MinionConstants.MergeTask.BUCKET_TIME_PERIOD_KEY));
+        Preconditions.checkState(bucketMs > 0, "Bucket time period must be larger than 0");
```

Comment:
Do we want to throw an exception here? Or, do we want to just log an error message and not generate any tasks and bump some metric that we can watch for non-zero?

Reply:
Thanks for pointing this out. I changed to emit an error log since we should continue to schedule for other tables (the current code will block other tables if one table has a bad configuration).

For metrics, we will eventually get the alerts from the delay metric so I don't think that we need to add another metric .